INSTALL.md

```diff
@@ -63,207 +63,277 @@ cfgov assets
+Run that now for your initial setup.
+Any time you run the project you’ll need to run that last line, so
+if you’ll be working with the project consistently,
+we suggest adding it to your Bash profile by running:
+
 ```bash
-echo 'source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
+echo 'source /Users/[YOUR MAC OS X USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
 ```
 
 If you need to find this info again later, you can run:
+
 ```bash
 brew info autoenv
 ```
 
-> **NOTE:** If you use ZSH you’ll need to use [zsh-autoenv](https://github.com/Tarrasch/zsh-autoenv),
+> **NOTE:** If you use Zsh you’ll need to use
+  [zsh-autoenv](https://github.com/Tarrasch/zsh-autoenv),
   but we can’t provide support for issues that may arise.
 
+#### MySQL
+
+If you're developing on OS X, this should be installed by default,
+and you shouldn't have to do anything else to get it working.
+You can optionally install a different version with Homebrew.
+
 #### Elasticsearch
 
 [Install Elasticsearch 1.7](https://www.elastic.co/guide/en/elasticsearch/reference/1.7/setup.html)
 however you’d like. (We use [Homebrew](http://brew.sh)):
+
 ```bash
 brew tap homebrew/versions
 brew search elasticsearch
 brew install homebrew/versions/elasticsearch17
 ```
 
-Just as with autoenv, Homebrew will output similar instructions after installation:
+Just as with Autoenv, Homebrew will output similar instructions after installation:
 
 ```bash
-To have launchd start homebrew/versions/elasticsearch17 now and restart at login:
+# To have launchd start homebrew/versions/elasticsearch17 now and restart at login:
   brew services start homebrew/versions/elasticsearch17
-Or, if you don't want/need a background service you can just run:
+# Or, if you don't want/need a background service you can just run:
   elasticsearch --config=/Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch17/config/elasticsearch.yml
 ```
 
-Any time you work on the project, you’ll need to open a new tab and run that last line.
-If you’ll be working on the project consistently, we suggest using the first option. Note that some older versions of Homebrew may suggest using `launchctl` instead of `brew services`.
+Any time you resume work on the project after restarting your machine,
+you’ll need to open a new tab and run that last line.
+If you’ll be working on the project consistently,
+we suggest using the first option, so you don't have to worry about that.
+Note that some older versions of Homebrew may suggest
+using `launchctl` instead of `brew services`.
 
 If you need to find this info again later, you can run:
+
 ```bash
 brew info elasticsearch17
 ```
 
-### MYSQL Database
-Start MYSQL with the following command:
-```
-mysql.server start
-```
+#### Front-end dependencies
 
-Then run the MYSQL creation script from project root directory:
-```
-./create-mysql-db.sh
-```
+The cfgov-refresh front end currently uses the following frameworks / tools:
```

Comment:
Though I grant that I may have been imperfect in attempting to abide by that rule.